darknet-training.dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM wildflowerschools/wf-deep-docker:cuda10.2-opencv-base-v0
+
+RUN apt install python3-venv -y && \
+    cd /build && git clone https://github.com/WildflowerSchools/darknet
+
+WORKDIR /build/darknet
+
+RUN sed -i 's/GPU=0/GPU=1/g' Makefile && \
```

Comment:
what is going on here?

Reply:
If I understand correctly you can do this:

`make GPU=1 CUDNN=1 CUDNN_HALF=1 OPENCV=1`

and get the same results

Reply:
Ah, interesting, I hadn't tried that approach. I followed an install guide on the darknet website that walks you through [editing the Makefile](https://pjreddie.com/darknet/install/), ergo the `sed` commands.